firedrake/mg/embedded.py

```diff
@@ -70,8 +70,11 @@ def __init__(self, *, native_transfers=None, use_averaging=True):
     def is_native(self, element, op):
         if element in self.native_transfers.keys():
             return self.native_transfers[element][op] is not None
-        if isinstance(element.cell, ufl.TensorProductCell) and len(element.sub_elements) > 0:
-            return all(self.is_native(e, op) for e in element.sub_elements)
+        if isinstance(element.cell, ufl.TensorProductCell):
```

Comment:
Why do we check the cell type here? Can't we have an `isinstance(element, TensorProductElement)` and `isinstance(element, MixedElement)`? (I know mixed elements are sometimes harder to identify than that)

Reply:
After running some multigrid tests, it turned out that `TensorProductCell` is of significance. So I tidied up a little, but made it sure that the change here is strictly noop.

Reply:
Weird, but not the point of this PR.